src/workerd/api/sockets.h

```diff
@@ -113,6 +116,25 @@ class Socket: public jsg::Object {
     JSG_METHOD(startTls);
   }
 
+  void visitForMemoryInfo(jsg::MemoryTracker& tracker) const {
+    tracker.trackFieldWithSize("connectionStream",
+        sizeof(IoOwn<kj::RefcountedWrapper<kj::Own<kj::AsyncIoStream>>>));
```

Comment:
Another question: would it work to say `sizeof(connectionStream)` here so you don't have to duplicate the type? If so, is there a reason to prefer explicitly writing the type here?

Reply:
It might. I prefer this approach since it's visually clearer what I'm measuring and that it's distinct from the other fields. But happy with whatever works.